code/api/src/seeders/1-user.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
         email: 'user@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.user,
+        description: 'Fashionista gurl',
```

Comment:
Thanks for adding in these descriptions for the existing users!

Reply:
this makes sense.  We will just need to update the photo and shipping address then right?

Reply:
So this user update endpoint will eventually be able to update all of the user fields: so email, image description, etc.
now that the endpoint is set up we just have to update the arguments going into the function, which we'll do later today

Reply:
right now it is set to just update the user image